hm_pyhelper/diagnostics/diagnostics_report.py

```diff
@@ -47,6 +52,21 @@ def append_error(self, key):
     def get_errors(self):
         return self[DIAGNOSTICS_ERRORS_KEY]
 
+    def has_manufacturing_errors(self) -> list:
+        # Check only a subset of keys that are required for manufacturing
+        # tests. If these don't have errors then we can conclude that device
+        # is good from manufacturing point-of-view.
+        errors = self.get_errors()
+
+        if errors:
+            self.set_passed(False)
```

Comment:
I still think this line should be removed. In practice, this should effectively result in a no-op because `passed` will already be false if there are any errors.

Reply:
Oh sorry about that I missed removing the line. Removed now.